src/navsat_transform_nodelet.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2014, 2015, 2016, Charles River Analytics, Inc.
```

Comment:
The package license is BSD, so we don't need to change that verbiage, but the copyright holder can be you or your organization. Note that I am not a copyright lawyer, but I'm pretty sure that's how it works. :)